src/main/webapp/resources/js/pages/projects/settings/components/ReferenceFiles.jsx

```diff
@@ -32,6 +35,12 @@ export default function ReferenceFiles() {
   );
   const [projectReferenceFiles, setProjectReferenceFiles] = React.useState([]);
   const [, setProgress] = React.useState(0);
+  const [total, setTotal] = React.useState(0);
+  const [paginationOptions, setPaginationOptions] = React.useState(null);
+
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(total);
}, [total]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b